pip/qsharp/env/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Environment module that receives any user-defined Q# callables as Python functions.
```

Comment:
Would a module level docstring do any good here?

Reply:
Added as part of changing the name from `env` to `code`